Tour/basic-types.md

```diff
@@ -46,41 +46,41 @@ Integer numbers are typically represented by the type `Int`, which represents a
 
 _Note: The type `Int` should be preferred unless you need a different variant for a specific reason (e.g., representing a hardware register, storage optimization)._ _This convention aids code consistency and interoperability._
 
-Floating point numbers are represented by the types `Float` and `Double`, denoting [IEEE](https://en.wikipedia.org/wiki/IEEE\_754) single and double-precision values respectively.
+Floating point numbers are represented by the types `Float32` and `Float64`, denoting [IEEE](https://en.wikipedia.org/wiki/IEEE\_754) single and double-precision values respectively.
 
-_Note: For the same reasons as `Int` should be preferred for every integer value, `Double` should be preferred for any floating-point value._
+_Note: For the same reasons as `Int` should be preferred for every integer value, `Float64` should be preferred for any floating-point value._
 
 Hylo does not support any kind of implicit conversion between numeric types. For example, the following program is illegal:
 
 ```
 public fun main() {
   let n = 3.2
   let m = 8
-  print(n * m) // error: cannot apply `Double.infix*` to argument of type `Int`
+  print(n * m) // error: cannot pass value of type 'Int' to parameter 'let Float64'
 }
 ```
 
-All numeric conversions must be written explicitly by calling the appropriate initializer. For example, we can fix the program above by converting `m` to `Double` before the multiplication:
+All numeric conversions must be written explicitly by calling the appropriate initializer. For example, we can fix the program above by converting `m` to `Float64` before the multiplication:
 
 ```
 public fun main() {
   let n = 3.2
   let m = 8
-  print(n * Double(m)) // 25.6
+  print(n * Float64(m)) // 25.6
```

Comment:
I removed the `print` and tested `(n * Float64(m))`, which throws:
```
.build/release/hc Examples/test.hylo -o test && ./test
Examples/test.hylo:4.7-14: error: ambiguous use of 'Float64'
  n * Float64(m) // 25.6
      ~~~~~~~
/workspaces/hylo/StandardLibrary/Sources/Core/Numbers/Floats/Float64.hylo:6.3-18: note: candidate here
  memberwise init
  ~~~~~~~~~~~~~~~
/workspaces/hylo/StandardLibrary/Sources/Core/Numbers/Floats/Float64.hylo:9.3-11:4: note: candidate here
  public init() {
  ~~~~~~~~~~~~~~~~
```
Is this expected?

Reply:
We miss a universal implementation of print. Once we have one this example will compile.